src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextFactoryTest.kt

```diff
@@ -102,7 +102,6 @@ class RecallNotificationContextFactoryTest {
     val lastReleasePrisonName = PrisonName("Last Prison Name")
     val prisoner = Prisoner(
       firstName = "Bobbie",
-      middleNames = "Bertrude",
```

Comment:
Not sure I'd delete this line; it's still valid input - both ways of course.

Reply:
Can see that, but in unit tests we generally only populate properties that are relevant to the test.